TOPICS.md

```diff
@@ -57,7 +58,17 @@ Entities can contain the following [point fields](https://docs.ros2.org/latest/a
 
 ## Freespace
 
-Freespace is provided as video frames. Black pixels (#000000) stand for non-freespace, while gray-blue (#434758 HEX) pixels mark freespace. The default coordinate system is as following:
+There are 2 ways of representing freespaces: [Polygonal](#polygonal-freespaces) and [Raw Images](#raw-image-freespaces). Please see below for more details.
+
+### Polygonal Freespaces
+
+The primary way of representing freespace is in the form of [polygons](ros/geometry_msgs/msg/PolygonInstanceStamped.idl). The polygons are represented as sequences of vertices defining their outlines. They can be both convex and concave. The "id" specifies the type of polygon, with 0 standing for freespaces.
```

Comment:
Just to double check, the "id" field in PolygonInstance would be equivalent to "Class" column in the polygon csv file right?

Reply:
@kevmcapro exactly, it's because the "id" one in csv was serving the purpose of distinguishing between points belonging to different polygons, and here it'll be done by sending multiple `PolygonInstanceStamped` messages instead. Unfortunately there is no multi-polygon message type in ROS2, and as usual I want to keep as compatible with it as possible. We can totally live with multiple `PolygonInstanceStamped` messages if required.